chainer/functions/noise/gaussian.py

```diff
@@ -19,10 +20,10 @@ class Gaussian(function_node.FunctionNode):
 
     """
 
-    def __init__(self):
+    def __init__(self, eps=None):
         # Per-instance noise that is generated once during its first forward
         # pass and then reused in subsequent calls, unless explicitly reset
```

Comment:
Maybe we should update this comment since it is not always generated.

Reply:
Fixed.